whois-api/src/main/java/net/ripe/db/whois/api/rdap/domain/RelationType.java

```diff
@@ -19,12 +21,21 @@ public String getValue() {
         return value;
     }
 
-    public static RelationType fromString(final String value) {
+    public static RelationType fromName(final String value) {
         for (RelationType type : RelationType.values()) {
             if (type.name().equalsIgnoreCase(value)) {
```

Comment:
Is `type.name()` going to match all values properly? For example, `RelationType.UP_ACTIVE.name()` is `UP_ACTIVE` not `up-active`. Do you mean to use `getValue()` instead?

Reply:
this is why I called fromName.
"up-active" doesn't match because the call should be .../top/...?status=active

getValue() is used in isValidValue for Integration Tests.

Reply:
Agreed to use `getValue()` and match value not enum name.